### PR TITLE
SPEC-182: Make it clearer how ASes are supposed to create users

### PR DESCRIPTION
**Pull request by Kegsay**
by linkifying the right text. (Also correctly linkify feature profiles!)

### CODE REVIEW

File: specification/3-application_service_api.rst [hunk 1]
```diff
@@ -94,7 +94,7 @@ API called when:
  - HS receives an event for an unknown user ID in the AS's namespace, e.g. an
    invite event to a room.
 Notes:
- - When the AS receives this request, if the user exists, it must create the user via
+ - When the AS receives this request, if the user exists, it must `create the user`_ via
    the CS API.
```
Comment (illicitonion) [inline specification/3-application_service_api.rst]: Also link to the CS API /createUser endpoint?

Comment (Kegsay) [inline specification/3-application_service_api.rst]: There is no api called that.

Comment (illicitonion) [inline specification/3-application_service_api.rst]: ... /register? Or whatever it's called?

Comment (Kegsay) [inline specification/3-application_service_api.rst]: I'm already linking to the register section for ASes about `m.login.application_service`

**Comment by illicitonion** [comment]
LGTM

**Pull request merged**